Sources/MapboxDirections/Intersection.swift

```diff
@@ -111,6 +113,12 @@ public struct Intersection {
      */
     public let restStop: RestStop?
 
+    /**
+     Street class according to Mapbox Streets V8 classification.
+     
+     This value is set to `nil` of such info is not available.
+     */
```

Comment:
```suggestion
    /**
     The road classes of the road that the containing step uses to leave the intersection, according to the [Mapbox Streets source](https://docs.mapbox.com/vector-tiles/reference/mapbox-streets-v8/#road), version 8.
     
     If detailed road class information is unavailable, this property is set to `nil`. This property only indicates the road classification; for other aspects of the road, use the `outletRoadClasses` property.
     */
```

And in the `outletRoadClasses` documentation, add a cross-reference like this:

> For more detailed road classifications, use the `outletMapboxStreetsRoadClass` property.